src/middleware.js

```diff
@@ -31,22 +31,22 @@ function init(swaggerPath, options) {
                     const parameters = dereferenced.paths[currentPath][currentMethod].parameters || [];
                     let bodySchema = parameters.filter(function (parameter) { return parameter.in === 'body' });
                     if (bodySchema.length > 0) {
-                        schemas[parsedPath][currentMethod].body = buildBodyValidation(bodySchema[0].schema, dereferenced.definitions, swaggers[1], currentPath, currentMethod, parsedPath);
+                        schemas[parsedPath][currentMethod].body = buildBodyValidation(bodySchema[0].schema, dereferenced.definitions, swaggers[1], currentPath, currentMethod, parsedPath, middlewareOptions.ajvConfigBody);
```

Comment:
you can manage those options the same as `middlewareOptions` in a global variable.
It will remove duplicate code and make the function signature cleaner.

Reply:
How would that work? Sub-functions would be accessing global ajvOptionsBody and ajvOptionsParam that are extracted from options entity?